src/main/java/com/rakovets/course/java/core/practice/xml/HibernateReader.java

```diff
@@ -0,0 +1,63 @@
+package com.rakovets.course.java.core.practice.xml;
+
+import javax.xml.stream.XMLEventReader;
+import javax.xml.stream.XMLInputFactory;
+import javax.xml.stream.XMLStreamException;
+import javax.xml.stream.events.Attribute;
+import javax.xml.stream.events.StartElement;
+import javax.xml.stream.events.XMLEvent;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.util.*;
+
+public class HibernateReader {
+    static LinkedHashMap<String, String> propertyMap = new LinkedHashMap<>();
+    static LinkedHashMap<String, String> mappingMap = new LinkedHashMap<>();
+
+    public HibernateConfiguration readConfig(String pathToXML) {
+        HibernateConfiguration Hibernate = new HibernateConfiguration();
```

Comment:
very bad name for variable.
```suggestion
        HibernateConfiguration hibernate = new HibernateConfiguration();
```